addons/mass_mailing_sms/static/tests/tours/mass_mailing_activities_split_test_tour.js

```diff
@@ -0,0 +1,45 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('mass_mailing_activities_split_test_tour', {
```

Comment:
Naming: 'mailing_activities_split' ? No need to repeat the 'tour' word, we already are in a tour directory